src/screens/MainUserPage/MainUserPage.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
```

Comment:
For this component, you could reuse the SuperOrgHome component as opposed to rebuilding an entire component from scratch
![image](https://user-images.githubusercontent.com/55003601/125209273-4442da80-e25d-11eb-9eed-fb04cce558ea.png)

Reply:
Sir I have not created this specific page.